<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalletConfigWidget</class>
 <widget class="QWidget" name="WalletConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>620</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin" stdset="0">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Wallet Preferences</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QCheckBox" name="_enabled">
         <property name="whatsThis">
          <string>&lt;p&gt;The wallet subsystem provides a convenient and secure way to manage all your passwords. This option specifies whether or not you want to use it.&lt;/p&gt;</string>
         </property>
         <property name="text">
          <string>&amp;Enable the KDE wallet subsystem</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KSeparator" name="kseparator"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox2">
         <property name="whatsThis">
          <string>It is best to close your wallets when you are not using them to prevent others from viewing or using them.</string>
         </property>
         <property name="title">
          <string>Close Behavior</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="labelAlignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="closeWalletLabel">
              <property name="text">
               <string>Close wallet:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="_autoclose">
              <property name="whatsThis">
               <string>&lt;p&gt;&lt;b&gt;Close wallet as soon as applications that use it have stopped.&lt;/b&gt;&lt;br&gt;Note that your wallet will only be closed when all applications that use it have stopped.&lt;br&gt;When a wallet is closed the password is needed to access it again.&lt;/p&gt;</string>
              </property>
              <property name="text">
               <string>When last application stops using it</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="_screensaverLock">
              <property name="whatsThis">
               <string>&lt;p&gt;&lt;b&gt;Close wallet as soon as the screensaver starts.&lt;/b&gt;&lt;br&gt;When a wallet is closed the password is needed to access it again.&lt;/p&gt;</string>
              </property>
              <property name="text">
               <string>When screensaver starts</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="_closeIdle">
              <property name="whatsThis">
               <string>&lt;p&gt;&lt;b&gt;Close wallet after a period of inactivity.&lt;/b&gt;&lt;br&gt;When a wallet is closed the password is needed to access it again.&lt;/p&gt;</string>
              </property>
              <property name="text">
               <string>When unused for:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="_idleTime">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="whatsThis">
               <string>&lt;p&gt;&lt;b&gt;Close wallet after a period of inactivity.&lt;/b&gt;&lt;br&gt;When a wallet is closed the password is needed to access it again.&lt;/p&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string> minutes</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox4">
         <property name="title">
          <string>Automatic Wallet Selection</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout3">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::FieldGrowthPolicy::ExpandingFieldsGrow</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
            </property>
            <item row="1" column="1">
             <widget class="QCheckBox" name="_localWalletSelected">
              <property name="text">
               <string>Use different default wallet for local passwords</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout4">
              <item>
               <widget class="QComboBox" name="_localWallet">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_newLocalWallet">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>New...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="selectDefaultWalletLabel">
              <property name="text">
               <string>Default wallet:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout3">
              <item>
               <widget class="QComboBox" name="_defaultWallet">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_newWallet">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>New...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox3">
         <property name="title">
          <string>Wallet Manager</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout2">
            <property name="formAlignment">
             <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
            </property>
            <item row="0" column="1">
             <widget class="QCheckBox" name="_launchManager">
              <property name="text">
               <string>Show manager in system tray</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="_autocloseManager">
                <property name="text">
                 <string>Hide system tray icon when last wallet closes</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox5">
         <property name="title">
          <string>Secret Service</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QFormLayout" name="formLayout4">
            <property name="formAlignment">
             <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
            </property>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout5">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="_secretServiceAPI">
                <property name="text">
                 <string>Use &amp;KWallet for the Secret Service interface</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="KContextualHelpButton" name="kcontextualhelpbutton">
                <property name="contextualHelpText">
                 <string>The Secret Service interface lets applications store passwords and other confidential data. Disable this to allow third-party wallet services such as KeePassXC or GNOME Keyring to manage it instead.</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Access Control</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="_openPrompt">
         <property name="text">
          <string>&amp;Prompt when an application accesses a wallet</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="_accessList">
         <property name="rootIsDecorated">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Wallet</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Application</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Policy</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="KSeparator" name="kseparator2"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="launchButtonBar">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_launch">
       <property name="text">
        <string>&amp;Launch Wallet Manager</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KContextualHelpButton</class>
   <extends>QToolButton</extends>
   <header>kcontextualhelpbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_localWalletSelected</sender>
   <signal>toggled(bool)</signal>
   <receiver>_newLocalWallet</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_localWalletSelected</sender>
   <signal>toggled(bool)</signal>
   <receiver>_localWallet</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_closeIdle</sender>
   <signal>toggled(bool)</signal>
   <receiver>_idleTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_closeIdle</sender>
   <signal>toggled(bool)</signal>
   <receiver>_idleTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_launchManager</sender>
   <signal>toggled(bool)</signal>
   <receiver>_autocloseManager</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
